plugins/auto-sizes/tests/test-improve-calculate-sizes.php

```diff
@@ -390,38 +390,147 @@ public function test_no_image(): void {
 	 * Test that the layout property of a group block is passed by context to the image block.
 	 *
 	 * @group test
+	 *
```

Comment:
Can you go ahead and remove the `@group test` ⬆️ annotation I added for ease of running just this test.